build/Targets/References.targets

```diff
@@ -1,25 +1,6 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <Choose>
```

Comment:
Can you add this file as a solution item in VS - so that it's easier to edit this file since i'm sure we'll keep updating dependencies.